man/madlib.svm.Rd

```diff
@@ -0,0 +1,294 @@
+  is a \code{svm.madlib} object. Otherwise, it is a \code{svm.madlib.grps}
+  object, which is just a list of \code{svm.madlib} objects.
+
+  A \code{svm.madlib} object is a list which contains the following items:
+
+  \item{coef}{
+    A vector, the fitting coefficients.
+  }
+
+  \item{grps}{
+    An integer, the number of groups that the data is divided into
+    according to the grouping columns in the formula.
+  }
+
+  \item{grp.cols}{
+    An array of strings. The column names of the grouping columns.
+  }
+
+  \item{has.intercept}{
+    A logical, whether the intercept is included in the fitting.
+  }
+
+  \item{ind.vars}{
+    An array of strings, all the different terms used as independent
+    variables in the fitting.
+  }
+
+  \item{ind.str}{
+    A string. The independent variables in an array format string.
+  }
+
+  \item{call}{
+    A language object. The function call that generates this result.
+  }
+
+  \item{col.name}{
+    An array of strings. The column names used in the fitting.
+  }
+
+  \item{appear}{
+    An array of strings, the same length as the number of independent
+    variables. The strings are used to print a clean result, especially when
+    we are dealing with the factor variables, where the dummy variable
+    names can be very long due to the inserting of a random string to
+    avoid naming conflicts, see \code{\link{as.factor,db.obj-method}}
+    for details. The list also contains \code{dummy} and \code{dummy.expr}, which are also used for processing the categorical variables, but do not contain any important information.
+  }
+
+  \item{model}{
+    A \code{\linkS4class{db.data.frame}} object, which wraps the model
+    table of this function.
+  }
+
+  \item{model.summary}{
+    A \code{\linkS4class{db.data.frame}} object, which wraps the summary
+    table of this function.
+  }
+
+  \item{model.random}{
+    A \code{\linkS4class{db.data.frame}} object, which wraps the kernel
+    table of this function. Only created when non-linear kernel is used.
+  }
+
+  \item{terms}{ A \code{\link{terms}} object, describing the terms in
+    the model formula.  }
+
+  \item{nobs}{
+    The number of observations used to fit the model.
+  }
+
+  \item{data}{
+    A \code{db.obj} object, which wraps all the data used in the database.
+    If there are fittings for multiple groups, then this is only the wrapper for the data in one group.
+  }
+
+  \item{origin.data}{
+    The original \code{db.obj} object. When there is no grouping, it is equal to \code{data} above, otherwise it is the "sum" of \code{data} from all groups.
+  }
+
+  Note that if there is grouping done, and there are multiple
+  \code{svm.madlib} objects in the final result, each one of them
+  contains the same copy \code{model}.
+}
+
+\author{
+  Author: Predictive Analytics Team at Pivotal Inc.
+
+  Maintainer: Frank McQuillan, Pivotal Inc. \email{fmcquillan@pivotal.io}
+}
+
+\references{
```

Comment:
There are some texts from the MADlib docs that use those references but I did not include them here to keep this concise. But I still include the full references since that is what other modules like `glm` and `elnet` also do.